NuGet.Config

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<packageSources>
+		<add key="NugetOfficialV3" value="https://api.nuget.org/v3/index.json" />
+		<add key="Local" value="tools/LocalNugetFeed" />
+	</packageSources>
+	
+	<config>
+		<add key="globalPackagesFolder" value="restoredPackages" />
```

Comment:
I'd like to avoid scoping to repo specific package cache.

Reply:
@weshaggard we have multiple repos that have multiple different versions of the same component as dependency.
Having a repo specific cache helps in isolating these dependencies into it's own cache.
Plus if your primarly concern is perf, this cache does not need to be deleted if you want the cache to be persistent.
I actually like this because this way I can test multiple repo at the same time.
e.g. Fluent.NET SDK, Multi-API repo and private repo, they all work independently and each of the repo has it's own separate cache.

Reply:
@weshaggard  now that I think about this, this is the least impactful things I can incorporate.
I will make this change now.